fsw/src/lc_msgdefs.h

```diff
@@ -35,29 +35,37 @@
 #include "lc_platform_cfg.h"
 
 /************************************************************************
- * Macro Definitions
+ * Type Definitions
  ************************************************************************/
 
 /**
  * \name LC Application States
  * \{
  */
-#define LC_STATE_ACTIVE   1 /**< \brief LC Application State Active */
-#define LC_STATE_PASSIVE  2 /**< \brief LC Application State Pasive */
-#define LC_STATE_DISABLED 3 /**< \brief LC Application State Disabled */
-#define LC_STATE_FROM_CDS 4 /**< \brief Used for reset processing, not valid state */
-/**\}*/
+typedef enum LC_AppState
+{
+    LC_STATE_ACTIVE = 1, /**< \brief LC Application State Active */
+    LC_STATE_PASSIVE,    /**< \brief LC Application State Pasive */
+    LC_STATE_DISABLED,   /**< \brief LC Application State Disabled */
+    LC_STATE_FROM_CDS    /**< \brief Used for reset processing, not valid state */
+} LC_AppState_t;
 
 /**
  * \name Actionpoint States
  * \{
  */
-#define LC_ACTION_NOT_USED  0xFF /**< \brief Actionpoint unused, not valid command argument */
-#define LC_APSTATE_ACTIVE   1    /**< \brief Actionpoint state active */
-#define LC_APSTATE_PASSIVE  2    /**< \brief Actionpoint state passive */
-#define LC_APSTATE_DISABLED 3    /**< \brief Actionpoint state disabled */
-#define LC_APSTATE_PERMOFF  4    /**< \brief Actionpoint state permanently off, see #LC_SET_AP_PERMOFF_CC */
-/**\}*/
+typedef enum LC_APState
+{
+    LC_APSTATE_ACTIVE = 1,   /**< \brief Actionpoint state active */
+    LC_APSTATE_PASSIVE,      /**< \brief Actionpoint state passive */
+    LC_APSTATE_DISABLED,     /**< \brief Actionpoint state disabled */
+    LC_APSTATE_PERMOFF,      /**< \brief Actionpoint state permanently off, see #LC_SET_AP_PERMOFF_CC */
+    LC_ACTION_NOT_USED = 255 /**< \brief Actionpoint unused, not valid command argument */
+} LC_APState_t;
```

Comment:
Similar typedef and name here (see cFE naming convention document)

Reply:
Updated.